mlir/include/mlir/Dialect/Transform/IR/TransformInterfaces.h

```diff
@@ -178,6 +184,11 @@ class TransformState {
     ParamMapping params;
     ValueMapping values;
     ValueMapping reverseValues;
+
+#ifndef NDEBUG
```

Comment:
Code that may change the size of the object should be guarded by `LLVM_ENABLE_ABI_BREAKING_CHECKS`.